nautobot/extras/templates/extras/metadatatype_create.html

```diff
@@ -0,0 +1,91 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load static %}
+
+{% block form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Metadata Type</strong></div>
+        <div class="panel-body">
+            {% render_field form.name %}
+            {% render_field form.description %}
+            {% render_field form.data_type %}
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Assignment</strong></div>
+        <div class="panel-body">
+            {% render_field form.content_types %}
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Choices</strong></div>
```

Comment:
It's not really clear that choices are only relevant for the "select" and "multiselect" data types. Perhaps this would help? I would prefer if this entire panel only showed up when those types are selected but may be out of scope for this issue.

```suggestion
        <div class="panel-heading"><strong>Choices (select and multiselect data types)</strong></div>
```